Tests/govuk_ios/govuk_ios_unit_tests/Specs/AppConfig/AppConfigServiceTests.swift

```diff
@@ -4,47 +4,104 @@ import XCTest
 
 final class AppConfigServiceTests: XCTestCase {
     
-    func test_isFeatureEnabled_whenFeatureFlagIsSetToAvilable_returnsTrue() throws {
+    func test_repository_isFeatureEnabled_whenFeatureFlagIsSetToAvailable_returnsTrue() throws {
         //Given
-        let mockAppConfigProvider = MockAppConfigProvider()
-        let sut = AppConfigService(configProvider: mockAppConfigProvider)
+        let mockAppConfigRepository = MockAppConfigRepository()
```

Comment:
This set up looks the same for all tests.  I'd be inclined to have it done in setup() and tearDown() methods instead.

Reply:
Done